src/python/pants/backend/python/subsystems/pytest_lockfile.txt

```diff
@@ -10,7 +10,7 @@
 #   ],
 #   "generated_with_requirements": [
 #     "pytest-cov!=2.12.1,<3.1,>=2.12",
-#     "pytest<6.3,>=6.2.4"
+#     "pytest<8,>=6.2.4"
```

Comment:
This needs to be regenerated. `build-support/bin/generate_all_lockfiles.sh --tool pytest`. No need to regenerate `--internal` though.